packages/destination-actions/src/destinations/adobe-target/index.ts

```diff
@@ -17,8 +17,24 @@ const destination: DestinationDefinition<Settings> = {
           'Your Adobe Target client code. To find your client code in Adobe Target, navigate to **Administration > Implementation**. The client code is shown at the top under Account Details.',
         type: 'string',
         required: true
+      },
+      bearer_token: {
+        label: 'Bearer Token',
+        description:
+          "Self managed bearer token generated via [Adobe's authentication API](https://developers.adobetarget.com/api/#authentication-tokens). Expires every 90 days.",
```

Comment:
@kdaswani What do you think of this description?

Reply:
Hi @marinhero ! I put a description in the ticket that I'd like to use: https://segment.atlassian.net/browse/STRATCONN-1670. It sounds like we may not want to explicitly put 90 days in the description though so here's my updated suggestion:

```suggestion
          "If you choose to require authentication for Adobe Target’s Profile API, you will need to generate an authentication token. Tokens can be generated in your Adobe Target account under the Implementation Settings tab or via the [Adobe.IO Authentication Token API](https://developers.adobetarget.com/api/#authentication-tokens). Input the authentication token here. Note: Authentication tokens expire so a new token will need to be generated and updated here prior to expiration.",

Reply:
Thank you @kdaswani. Really appreciate the help here!